data/solar_farm_data/april/solardata_minutes_4_2016.csv

```diff
@@ -0,0 +1,2881 @@
+time,measured,inverted
```

Comment:
Is this raw data parsed from what utilities sent, or was this generated by one of the scripts here? There needs to be a better naming convention for this data and more metadata regarding its origin (ideally, in a well-organized single authoritative location...)

Also, naming should never end with the year --- If this is April 2016 data, then the name should be `2016_04_stuff`, not `stuff_4_2016`. OR, even better, use the ISO standard (https://docs.python.org/2/library/datetime.html#datetime.date.isoformat). 

This is so that files are well ordered (also, you don't need to put this file in a folder called april.. because that information is already contained in the name).

I recommend reading the book, Clean Code by Robert C. Martin. It has a great chapter on naming things.